uber/models.py

```diff
@@ -707,6 +724,14 @@ def age_group_desc(self):
             return session.age_group_from_birthdate(self.birthdate).desc
 
     @property
+    def watchlist_entry(self):
```

Comment:
The Watch List stuff either belongs in its own plugin or in master. Not sure which.